setup.ts

```diff
@@ -418,7 +409,7 @@ async function main(): Promise<void> {
     refreshToken: string | null = "";
   if (process.env.ACCESS_TOKEN_SECRET) {
     console.log(
-      `\nAccess token secret already exists with the value:\n${process.env.ACCESS_TOKEN_SECRET}`
+      `\nAccess token secret already exists with the value: \n${process.env.ACCESS_TOKEN_SECRET}`
```

Comment:
In all cases for all values, if the value already exists, then the default action for creating a new value must be "No", not "Yes"

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/16875803/d1bd4aa4-a560-4a43-97ab-7413f9280169)